src/coreclr/interpreter/compiler.cpp

```diff
@@ -5085,9 +5042,7 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                         break;
                     }
                     default:
-                        assert(!"Unsupported element type for STELEM");
-                        m_hasInvalidCode = true;
-                        goto exit_bad_code;
+                        BADCODE("Unsupported element type for STELEM");
```

Comment:
Is this unsupported in a sense that the other element types than the one we currently handle are invalid or in the sense that we don't support it yet?

Reply:
I don't know. I'm just replacing the `m_hasInvalidCode` logic, not changing behavior.